www/content/tutorials/graphql/enable-modules.mdx

```diff
@@ -8,13 +8,20 @@ group: Quick Start (GraphQL)
 1. Visit `/admin/modules`.
 2. Enable the following modules: **Next.js** and **Next.js GraphQL**.
 
+## Configure GraphQL Compose
+
+1. Visit `/admin/config/graphql_compose`.
+2. Enable "GraphQL", "Single Query", "Edge Query", & "Loading by Route" for the Article content type.
```

Comment:
"Edge Query" is not an option unless "GraphQL Compose: Edges" is enabled, which the instructions do not indicated we need to do.

"Loading by Route" is not an option unless "GraphQL Compose: Routes" is enabled, which the instructions do not indicated we need to do.

So… we need to update the step above "2. Enable the following modules" to include those 2 modules. Are there other modules that should also be enabled?

Reply:
You're right, I forgot the module was split up a bit between beta and 2.x releases.  I'll try to get a commit in for this this week if I can clear some time.

Reply:
@JohnAlbin it took a while for me to get back to it, but I've updated the list of modules for the GrpahQL example docs.